app/routes/errors/HTTPError.js

```diff
@@ -6,11 +6,17 @@ import { Link } from 'react-router-dom';
 import renderAbakus from './renderAbakus';
 
 const HTTPMapping = {
-  '400': 'Noe gikk galt med forespørselen',
-  '401': 'Du er ikke logget inn',
-  '403': 'Denne siden har du ikke tilgang på',
-  '404': 'Denne siden finnes ikke',
-  '500': 'Noe gikk veldig galt, Webkom er på saken!',
+  // Non-string literal property keys is not supported in flow yet
```

Comment:
Same here, is there a reason?

Reply:
Not an issue in the latest version of eslint.